Update Pontoon documentation resources.md (#209)
mozilla-l10n/localizer-documentation: src/tools/pontoon/resources.md

<!-- toc -->

Pontoon features built-in access to different translation memories. These resources are available in the translation workspace. Machinery may also be accessed via the navigation bar as a standalone web page.

## History

> Pontoon displays the translation history for strings below the editor. The current approved translation for a source string appears at the top and all previous suggestions and translations beneath.

Reviewer: We also explain this (in slightly more detail) in line 29. Can we bring everything here?

Author: This part is laying out how the editor/translation history appear in Pontoon, while line 29 covers how the DIFF function works. I think the current order makes sense to introduce parts of translation history before going into detail on DIFF.

![History tab](../../assets/images/pontoon/resources/history.png)

Each translation history entry contains:
* The name of the translator and their profile picture (linking to their profile).
* How long ago the entry was submitted (hover over to see the full date and time as a tooltip).

> * The translation or suggestion.

Reviewer: Maybe just say "Translation"? https://github.com/mozilla-l10n/localizer-documentation/pull/213/files#diff-015370b13beb016806b36c39ecc5b6f21cfca5e9d63716f84c7aa563d5614bb2R29

* If both icons are gray, translation has been suggested but not yet reviewed.
* The red cross indicates that the translation has been rejected. The entire element will look opaque.
* The trashcan, available only for rejected translations, can be used to completely delete a translation.

### Viewing Translation Differences

The `DIFF` option appears if there are multiple suggestions for one string. Toggling `DIFF` compares the selected suggestion to the top entry, either the current approved translation or the most recent suggestion if no translation has been approved. Text highlighted in green indicates the text that has been added (compared to the top entry), text in red indicates removed text. Toggling `DIFF` again will display the original string.

![Diff for suggestions](../../assets/images/pontoon/resources/suggestions_diff.png)

## Machinery tab

The Machinery tab in the translation view shows possible translations from a variety of sources. These sources include:
* [Pontoon’s internal translation memory](translate.md#downloading-and-uploading-translations).
* [Microsoft Terminology](https://www.microsoft.com/Language/).
* [Google Translate](https://translate.google.com).

> In addition, the user has the ability to search for translations containing words via [`Concordance Search`](#concordance-search)

Reviewer: Missing period.

> In the tab, the number of available entries is visible alongside the `MACHINERY` title.

Reviewer: Since the screenshot shows "2+1", we should clarify what that means.

* If both icons are gray, translation has been suggested but not yet reviewed.
* The red cross indicates that the translation has been rejected. The entire element will look opaque.
* The trashcan, available only for rejected translations, can be used to completely delete a translation.

### Viewing Translation Differences

The `DIFF` option appears if there are multiple suggestions for one string. Toggling `DIFF` compares the selected suggestion to the top entry, either the current approved translation or the most recent suggestion if no translation has been approved. Text highlighted in green indicates the text that has been added (compared to the top entry), text in red indicates removed text. Toggling `DIFF` again will display the original string.

![Diff for suggestions](../../assets/images/pontoon/resources/suggestions_diff.png)

## Machinery tab

The Machinery tab in the translation view shows possible translations from a variety of sources. These sources include:
* [Pontoon’s internal translation memory](translate.md#downloading-and-uploading-translations).
* [Microsoft Terminology](https://www.microsoft.com/Language/).
* [Google Translate](https://translate.google.com).

> In addition, the user has the ability to search for translations containing words via [`Concordance Search`](#concordance-search).

Reviewer 2: ```suggestion
In addition, the user has the ability to search for translations containing words via [`Concordance search`](#concordance-search).
```

You used lowercase later.

To the right of the entry, the number in green shows the percent match between the machinery string and the source string. The higher the percentage, the more likely the machinery is to be useful. The machinery source is listed in gray. Upon clicking the gray text, the machinery source will be opened in a new window. The green superscript to the right indicates the number of matches for the entry in the translation memory source.

Pay careful attention when using the machinery tab as suggestions may not match the source string for the project being translated. Even if the source strings match, the context in which strings are used may not be the same. This could lead to incorrect translation or unnatural translations in the context of the project being translated. Always keep the meaning and purpose of the string being translated in mind when using the machinery tab.

### Concordance search

> Concordance search allows users to search across all projects in Pontoon. Users can search for translations using strings in either source or target language. Matching results are displayed with the source string, translation, and project name and clicking a result will automatically fill the translation into the editing window. The search does not need to be related to the current project string.

Reviewer 2: ```suggestion
Concordance search allows users to search across all projects in Pontoon. Users can search for translations using strings in either source or target language. Matching results are displayed with the source string, translation, and project name; clicking a result will automatically fill the translation into the editor. Note that the search does not need to be related to the current string or project.
```

* If both icons are gray, translation has been suggested but not yet reviewed.
* The red cross indicates that the translation has been rejected. The entire element will look opaque.
* The trashcan, available only for rejected translations, can be used to completely delete a translation.

### Viewing translation differences

The `DIFF` option appears if there are multiple translations for one string. Toggling `DIFF` compares the text to the current approved translation, or the most recent suggestion if no translation has been approved yet. Text highlighted in green indicates content that has been added, while strikethrough text in red indicates removed content. Toggling `DIFF` again will display the original string.

![Diff for suggestions](../../assets/images/pontoon/resources/suggestions_diff.png)

## Machinery tab

The Machinery tab in the translation view shows possible translations from a variety of sources. These sources include:
* [Pontoon’s internal translation memory](translate.md#downloading-and-uploading-translations).
* [Microsoft Terminology](https://www.microsoft.com/Language/).
* [Google Translate](https://translate.google.com).

> * [SYSTRAN](https://www.systran.net/)
> * [Caighdean](https://github.com/kscanne/caighdean)
> * [Bing Translator](https://www.bing.com/translator) (not currently enabled on pontoon.mozilla.org)

Reviewer: ```suggestion
* [SYSTRAN](https://www.systran.net/).
* [Caighdean](https://github.com/kscanne/caighdean).
* [Bing Translator](https://www.bing.com/translator) (not currently enabled on pontoon.mozilla.org).
```

Be careful when using the machinery tab as suggestions may not match the source string for the project being translated. Even if the source strings match, the context in which strings are used may not be the same. This could lead to incorrect or unnatural translations. Always keep the meaning and purpose of the string being translated in mind when using the machinery tab.

### Concordance search

Concordance search allows users to search across all projects in Pontoon. Users can search for translations using strings in either source or target language. Matching results are displayed with the source string, translation, and project name; clicking a result will automatically fill the translation into the editor. Note that the search does not need to be related to the current string or project.

![Concordance search](../../assets/images/pontoon/resources/concordance_search.png)

### Stand-alone machinery

> Pontoon machinery is also available as a [stand-alone web page](https://pontoon.mozilla.org/machinery/). To access the page, use the `Machinery` link available in the Profile menu, or the `Machinery` tab in the upper navigation displayed in some pages (e.g. *Teams page* or *Projects page*).

Reviewer: ```suggestion
Pontoon machinery is also available as a [stand-alone web page](https://pontoon.mozilla.org/machinery/). To access the page, use the `Machinery` link in the main navigation (not accessible from the translation workspace).
```

To use the machinery, select the target locale from the dropdown menu to the right of the searchbar. Then input the English source string into the searchbar and press Enter. This will display all matching machinery entries. Information is organized exactly as in the translation workspace.

## Locales tab

The locales tab shows approved translations from Pontoon projects in other [locales](glossary.md#locale).

![Locales tab](../../assets/images/pontoon/resources/locales.png)

> Next to the `LOCALES` title, the number of available entries is visible. The number of preferred locales are in green, all other locales are listed in gray. 

Reviewer: ```suggestion
Next to the `LOCALES` title, the number of available entries is visible. The number of preferred locales are in green, all other locales are listed in gray.
```

Reviewer: Curious - since it's just **one** "number of preferred locales", why **are**?

Pontoon features built-in access to different translation memories. These resources are available in the translation workspace and, for machinery, on a [standalone web page](https://pontoon.mozilla.org/machinery/).

## History

Pontoon displays the translation history for strings below the editor. The current approved translation for a source string appears at the top and all previous suggestions and translations beneath.

![History tab](../../assets/images/pontoon/resources/history.png)

Each translation history entry contains:
* The name of the translator and their profile picture (linking to their profile).

> * How long ago the entry was submitted (hover over to see the full date and time is available as a tooltip).

Reviewer 2: The sentence between parentheses is broken, the second part likely needs removing

* The translation or suggestion.
* Icons indicating translation status (see below).

> * Translation comments. You can toggle `COMMENT` to show all comments for a translation or to add a new comment.

Reviewer 2: I think we can differentiate this. There will be the number of comments displayed, if there are any.

Icons to the right indicate the status of each translation:

> * The green checkmark indicates that the translation has been approved.

Reviewer 2: Missing status for pretranslation.

* If both icons are gray, translation has been suggested but not reviewed yet.
* The red cross indicates that the translation has been rejected. The entire element will look opaque.
* The trashcan, available only for rejected translations, can be used to completely delete a translation.

### Viewing Translation Differences

> The `DIFF` option appears if there are multiple suggestions for one string. Toggling `DIFF` compares the selected suggestion to the top entry, either the current approved translation or the most recent suggestion if no translation has been approved. Text hilighted in green indicates the text that has been added (compared to the top entry), text in red indicates removed text. Toggling `DIFF` again will display the original string.

Reviewer 2: Typo: hilighted

![Diff for suggestions](../../assets/images/pontoon/resources/suggestions_diff.png)

## Machinery tab

The Machinery tab shows possible translations from a variety of sources. These sources include:

> * [Pontoon’s internal translation memory](../pontoon/translate.md#downloading-and-uploading-translations).

Reviewer 2: ```suggestion
* [Pontoon’s internal translation memory](translate.md#downloading-and-uploading-translations).
```

Files are in the same folder.

> To the right of the entry, the number in green shows the percent match between the machinery string and the source string. The higher the percentage, the more likely the machinery is to be useful. The machinery source is listed in gray. Upon clicking the gray text, the machinery source will be opened in a new window. The green superscript to the right indicates the number of matches for the entry in the translation memory source.

Reviewer: The two paragraphs above no longer match the screenshots. The first one probably needs to be removed, because Machinery search has been replaced with Concordance search, which is covered below.
